### ISSUE TITLE
Uncaught TypeError: Cannot read properties of null (reading 'response')

### ISSUE THREAD

**Issue by nathanredblur** [Leanplum/Leanplum-JavaScript-SDK#146]
## Expected Behavior

No Errors

## Actual Behavior
I got
`Uncaught TypeError: Cannot read properties of null (reading 'response')`
![image](https://user-images.githubusercontent.com/1273018/149540707-ed414cf2-7941-4def-8423-b78adabe11ea.png)

```
Uncaught TypeError: Cannot read properties of null (reading 'response')
    at response (leanplum.min.js?98de:formatted:1737)
    at eval (leanplum.min.js?98de:formatted:216)
```
  | Function | @ | location
-- | -- | -- | --
  | response | @ | leanplum.min.js?98de:formatted:1737
  | eval | @ | leanplum.min.js?98de:formatted:216
  | setTimeout (async) |   |  
  | c.onreadystatechange | @ | leanplum.min.js?98de:formatted:216
  | XMLHttpRequest.send (async) |   |  
  | s.ajax | @ | leanplum.min.js?98de:formatted:234
  | f | @ | leanplum.min.js?98de:formatted:1957
  | a.request | @ | leanplum.min.js?98de:formatted:1965
  | y.createRequest | @ | leanplum.min.js?98de:formatted:1499
  | a.downloadMessages | @ | leanplum.min.js?98de:formatted:1730
  | fetchNotifications | @ | leanplum.js?2c61:143
  | eval | @ | leanplum.js?2c61:136
  | setInterval (async) |   |  
  | leanplumInboxSubscribe | @ | leanplum.js?2c61:135
  | _callee2$ | @ | useLeanplum.js?ee6e:64

## Steps to Reproduce the Problem

Looks like happen when a request to leamplum API is not successful or
when unmount the component that consume leanplum.

## Specifications

  - Version: "leanplum-sdk": "^1.3.0",
  - Platform: web
  - Subsystem: n/a

**Comment by gyoshev** [comment]
Hi! This should be resolved by e72914476da680f74be83294f8d1dff3044b27db in version 1.6.0 and later.

**Comment by nathanredblur** [comment]
Wonderful. Thank you.
